src/kehaar_component/external_service.clj

```diff
@@ -0,0 +1,54 @@
+(ns kehaar-component.external-service
+  (:require
+    [clojure.core.async :as async]
+    [com.stuartsierra.component :as component]
+    [kehaar.wire-up :as wire-up]
+    [langohr.core :as rmq]))
+
+;; Pseudo spec/schema just so it's clear what we expect for config
+;; here--could later be used to validate config with some adjustments.
```

Comment:
So, the only reason I'm hesitant to add this right now is that `clojure.spec` is right around the corner, and I'd rather wait until that is out before bothering to incorporate this via schema.